index.html

```diff
@@ -13,39 +13,239 @@
   </head>
 
   <body>
-    <div>
-      <header class="header">
-        <nav class="nav">
-          <a href="">Welcome</a>
-          <i class="fas fa-bars"></i>
-        </nav>
-      </header>
-      <section class="hero-heading">
-        <h1 class="title">Hey there, I am Dennis.</h1>
-        <p id="title">I'm a software Developer</p>
-        <p class="description">
-          I can help you build a product, feature or website Look through some
+    <header class="header">
+      <nav class="nav">
+        <a href="">Welcome</a>
+        <i class="fas fa-bars"></i>
```

Comment:
- [ ] **Required** Let move the hamburuger to the top-right edge of the design. Remember, the design must be similar to the [figma sample design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468).

**Your design** 

![image](https://user-images.githubusercontent.com/27709832/143660219-22ea1474-bf90-4560-9cde-3eae97164fcd.png)

**Fima design** 

![image](https://user-images.githubusercontent.com/27709832/143660199-0aab16f7-dc06-475b-8837-cba582ee0a31.png)

Reply:
Ahhh...this was as a result of removing the div holding the header; the first reviewer requested I remove and I didn't check the behaviour after removing it. However, instead of div, I have wrapped it inside of the section and updated the CSS. 

Thank you.